music21/configure.py

```diff
@@ -1218,7 +1218,8 @@ def _getDarwinApp(self, comparisonFunction) -> List[str]:
         This looks at everything in Applications, as well as every directory in Applications
         '''
         post: List[str] = []
-        for path0 in ('/Applications', common.cleanpath('~/Applications')):
+        for path0 in ('/Applications', common.cleanpath('~/Applications', returnPathlib=False)):
+            assert isinstance(path0, str)
```

Comment:
Likewise, `cleanpath(returnPathlib=False)` should always return a string, but if someday it doesn't there's nothing a user can do here.